### PR TITLE
CS-43661-added developerhub url from cma, and version bump

### PR DESCRIPTION
**Pull request by cs-raj**
version bump:
@contentstack/apps-cli: 1.0.6 -> 1.0.7

### CODE REVIEW

File: src/util/inquirer.ts [hunk 1]
```diff
@@ -159,15 +158,15 @@ async function getInstalledApps(
...
   let developerHubBaseUrl = cma.replace('api','developerhub-api')
-
-  developerHubBaseUrl = developerHubBaseUrl.startsWith('dev9')?developerHubBaseUrl.replace('dev9','dev'):developerHubBaseUrl;
   
   if (developerHubBaseUrl.startsWith("http")) {
     developerHubBaseUrl = developerHubBaseUrl.split("//")[1];
   }
 
+  developerHubBaseUrl = developerHubBaseUrl.startsWith('dev11')?developerHubBaseUrl.replace('dev11','dev'):developerHubBaseUrl;
+  console.log(developerHubBaseUrl)
```
Comment (aman19K) [inline src/util/inquirer.ts:169]: @cs-raj remove console log if not required.

**Review by aman19K — APPROVED** [review]

File: src/util/inquirer.ts (hunk 1 above)
Comment (cs-raj) [reply src/util/inquirer.ts]: Removed

File: src/util/inquirer.ts [hunk 2]
```diff
@@ -159,28 +158,14 @@ async function getInstalledApps(
...
-  }
-
+  const { cma } = configHandler.get("region") || {};
+  let developerHubBaseUrl = cma.replace('api','developerhub-api')
+  
   if (developerHubBaseUrl.startsWith("http")) {
     developerHubBaseUrl = developerHubBaseUrl.split("//")[1];
   }
 
+  developerHubBaseUrl = developerHubBaseUrl.startsWith('dev11')?developerHubBaseUrl.replace('dev11','dev'):developerHubBaseUrl;
```
Comment (antonyagustine) [inline src/util/inquirer.ts:168]: Can you check the code format/alignment?

Comment (cs-raj) [reply src/util/inquirer.ts]: Changed

**Review by antonyagustine — APPROVED** [review]

**Review by aman19K — APPROVED** [review]

**Pull request merged**